.github/workflows/go.yml

```diff
@@ -16,8 +19,9 @@ jobs:
         go-version: 1.14
       id: go
 
-    - name: install mingw
-      run : sudo apt-get install -y gcc-mingw-w64
+    - name: --Linux-- Install mingw
+      if: ${{ runner.os == 'Linux' }}
+      run: sudo apt-get install -y gcc-mingw-w64
```

Comment:
Don't think we need this now? This was to enable cross builds from Linux to Windows.

Reply:
Oh then I'll remove it. Reason it wasn't needed on windows is that the windows image comes with it pre-installed.